src/renderer/components/ProjectCreation/CloudConfigView.tsx

```diff
@@ -0,0 +1,168 @@
+import ipc from '../../ipc';
+
+const { openExternalLink } = ipc;
+
+interface Props {
+  prevView: () => void;
+  closeModal: () => void;
+  nextView: () => void;
+  projectName: string;
+  textToDisplay: string | null;
+}
+
+const CustomStack = styled(Stack)({ width: '100%' });
+
+const CustomColumnStack = styled(CustomStack)({ flexDirection: 'column' });
+
+const CustomRowStack = styled(CustomStack)({
+  flexDirection: 'row',
+  alignItems: 'center',
+});
+
+const Container = styled(Box)({
+  backgroundColor: colors.grey[700],
+});
+
+const CloudConfigView = ({
+  prevView,
+  closeModal,
+  nextView,
+  projectName,
+  textToDisplay,
+}: Props) => {
+  const [isAwaitingAPIKey, setAwaitingAPIKey] = useState<boolean>(true);
+  const [isAwaitingClientSecret, setAwaitingClientSecret] =
+    useState<boolean>(true);
+  const [APIKey, setAPIKey] = useState<string>('');
+  const [clientSecret, setClientSecret] = useState<string>('');
+
+  const saveCloudCredentials: () => void = () => {
+    setAPIKey(APIKey.trim());
+    setClientSecret(clientSecret.trim());
+    nextView();
+  };
+
+  const handleAPIKeyInput = (
+    event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    const key = event.target.value;
+    setAPIKey(key);
+    setAwaitingAPIKey(key.length === 0);
+  };
+
+  const handleClientSecretInput = (
+    event: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => {
+    const secret = event.target.value;
+    setClientSecret(secret);
+    setAwaitingClientSecret(secret.length === 0);
+  };
+
+  const saveButton = (
+    <PrimaryButton
+      onClick={saveCloudCredentials}
+      disabled={isAwaitingAPIKey || isAwaitingClientSecret}
+      fullWidth
+    >
+      Save
+    </PrimaryButton>
+  );
+
+  const cancelButton = (
+    <SecondaryButton onClick={prevView} fullWidth>
+      Back
+    </SecondaryButton>
+  );
+
+  const handleHelpClick: () => void = () => {
+    openExternalLink('https://mlvet.app');
+  };
+
+  const defaultText =
+    "This is your first time using the cloud transcription method. To get started you'll need to provide an API key and client secret for Google Cloud Speech-to-Text";
+
+  const text = textToDisplay ?? defaultText;
+
+  useKeypress(
+    saveCloudCredentials,
+    !(isAwaitingAPIKey || isAwaitingClientSecret),
+    ['Enter', 'NumpadEnter']
+  );
+
+  return (
+    <Container sx={{ height: { xs: 500 } }}>
+      <CustomRowStack justifyContent="space-between">
+        <Typography
+          overflow="hidden"
+          textOverflow="ellipsis"
+          variant="h1"
+          sx={{ color: colors.grey[400] }}
+        >
+          {projectName}
+        </Typography>
+        <IconButton
+          sx={{ color: colors.yellow[500], fontSize: 20 }}
+          onClick={closeModal}
+        >
+          <CloseIcon />
+        </IconButton>
+      </CustomRowStack>
+      <CustomColumnStack
+        justifyContent="space-between"
+        sx={{ height: '90.5%' }}
```

Comment:
Bit of a magic number / hacky - why so specific?

Reply:
Yeah it is hacky, I couldnt get it right

Reply:
Im going to re-visit this when I re-adjust the padding, etc. that Riley mentioned